LongWarOfTheChosen/Localization/XComGame.rus

```diff
@@ -1032,26 +1035,30 @@ BriefSummary="Увеличивает меткость на <Ability:SCOPE_SUP_AI
 TinySummary="Повышенная меткость при обычных выстрелах"
 LootTooltip="Это улучшение повысит меткость любого оружия, на которое оно будет установлено, при обычной (не ответной) стрельбе."
 
+
+LWOTC Translated
 [MissDamageUpgrade_Bsc X2WeaponUpgradeTemplate]
-FriendlyName="Приклад"
-FriendlyNamePlural="Приклады"
-BriefSummary="Дает способность \"Устойчивая позиция\", добавляющую +<Ability:STOCK_BSC_SW_AIM_BONUS/> к меткости и +<Ability:STOCK_BSC_SW_AIM_BONUS/> к вероятности критического попадания при следующем выстреле."
-TinySummary="Дает способность \"Устойчивая позиция\""
-LootTooltip="Это улучшение дает солдату способность \"Устойчивая позиция\"."
+FriendlyName="Рукоятка"
+FriendlyNamePlural="Рукоятки"
+BriefSummary="Промахи из основого оружия с 20% шансом могут стать царапиной."
```

Comment:
"20%" should be ` <Ability:STOCK_BSC_GF_CHANCE/>%` instead. Also note that the basic Stock provides a 25% chance of Grazing Fire, not 20%. And it may change in the future, which is why you should use the ability tag (that's what the `<Ability:...` thing is called) instead.

Reply:
Actually, I realise you're translating the content on the `master` branch, so it's understandable you used the hard-coded "20%". In order to ensure your changes aren't out of date as soon as we create beta 3, please see [the content here](https://github.com/long-war-2/lwotc/blob/experimental/LongWarOfTheChosen/Localization/XComGame.int#L1180) for the current English localisation text.

Reply:
Done.
Okay, i`ll try not to forget check versions changelist)